agent/bench-scripts/pbench-fio.md

```diff
@@ -233,9 +233,9 @@ Then you construct a fio job file for your initial sequential tests, and this wi
 
 And you write it to a file named fio-sequential.job, then run it with a command like this one, which launches fio on 1K guests with json output format.
 
-    /usr/local/bin/fio --client-file=vms.list --pre-iteration-script=drop-cache.sh \
-       --rw=write,read -b 4,128,1024 -d /mnt/fio/files --max-jobs=1024 \
-       --output-format=json fio-sequential.job
+    pbench-fio --client-file=vms.list --pre-iteration-script=drop-cache.sh \
+       -t write,read -b 4,128,1024 -d /mnt/fio/files --numjobs=32 \
+       --job-file fio-sequential.job
```

Comment:
Why make all these changes when the comments at line 234 and 240 don't match the new changes?

If the original line was supposed to work with the base `fio` program itself, then we should fix it.  If it is intended to be run with `pbench-fio` wrapper as this new change suggests, we should consider updating the text surrounding the command to reflect the intention.

Reply:
fio doesn't take the parameters --client-file, --pre-iteration-script.
And pbench-fio does not take the parameters --output-format=json, --max-jobs --rw. So as such this command will neither work for fio nor pbench-fio. I'll change the texts around it.